examples/table/table/app.py

```diff
@@ -19,8 +19,12 @@
 
 class ExampleTableApp(toga.App):
     # Table callback functions
-    def on_select_handler(self, widget, row, **kwargs):
-        self.label.text = 'You selected row: {}'.format(row.title) if row is not None else 'No row selected'
+    def on_select_handler1(self, widget, row, **kwargs):
+        self.label_table1.text = 'You selected row: {}'.format(row.title) if row is not None else 'No row selected'
+
+    def on_select_handler2(self, widget, row, **kwargs):
+        self.label_table2.text = 'You selected rows: {}'.format(','.join([r.title for r in row])) \
+                                 if row is not None else 'No row selected'
```

Comment:
These lines are long enough that they're difficult to read; switching to classic "if/else" syntax rather than trying to do this in one line would be preferable.

Also; two issues with multi-select handler specifically:
1. When you select multiple rows, you rapidly run out of space. The label should probably only indicate the number of rows that are selected, and/or the label of the most recently selected row.
2. `row` will always be the row that was *just* selected, not an iterable.

Reply:
1. Ok.
2. So, if `row` it's the last row selected, then to get all rows selected I have to create an attribute of the table like for ex.: `selected` or `selected_rows` right?